.github/workflows/code-quality.yml

```diff
@@ -28,10 +27,3 @@ jobs:
     uses: ./.github/workflows/static-analysis.yml
     secrets:
       COMPOSER_AUTH_JSON: ${{ secrets.COMPOSER_AUTH_JSON }}
-
-  slic:
-    name: 'PHP Tests'
-    needs: [coding-standards, phpstan, linting]
-    uses: ./.github/workflows/php-tests.yml
-    secrets:
-      COMPOSER_AUTH_JSON: ${{ secrets.COMPOSER_AUTH_JSON }}
```

Comment:
I wonder if we should comment this out rather than remove it entirely?

Reply:
I thought about that. I'm fine with doing that if we're ok committing commented code.

Reply:
In this case, I think it make sense as it may be used in the future.

Reply:
Updated